etl/src/main/java/org/pih/hivmigration/etl/sql/Setup.groovy

```diff
@@ -75,7 +75,8 @@ class Setup extends SqlMigrator {
                 field_name VARCHAR(100),
                 field_value VARCHAR(1000),
                 warning_type VARCHAR(255),
-                warning_details VARCHAR(1000)
+                warning_details VARCHAR(1000),
+                flag_for_review BOOLEAN DEFAULT FALSE
```

Comment:
Maybe it makes more sense to make flag_for_review DEFAULT TRUE, so you won't have to update all those insert statements above. I think that by default any row insert into this table should be reviewed. The use cases where this flag would be FALSE are less likely and then you could just specify it in the insert statement.

Reply:
Thanks @cioan !  My thought for this column was to distinguish data that was "ready for review" by a data clerk vs stuff that still needed review from us, so defaulting it to false was intended so that we would have to "manually" flag something for review by the team.

I wasn't intending for this column at all, but as I was ready to move the this ticket into test I wanted to ask Debbie to assign it to Ricard to review, and realized that there were many rows that it really didn't make sense to ask MEQ or data clerks to review yet... for example there were 3,199 "invalid" CD4 counts row, and I figured with that many we should see if there's any automated cleanup we could do first.  Then there were ones like "cd4 value belongs to an encounter that was not migrated. Value not migrated." which were really more warning for us than data clerks.  My intent was to use this field to distinguish those two types.  Hope that makes sense!